src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    operate(func, value) {
```

Comment:
How the calculator will work:
 * - Each `operate` call takes a callback and a number and **sets the
 *   appropriate value to the `result` property.**
in your case the result property is changing in methods: `add`, `subtract`, `multiply`, `divide`, but not in 'operate' method